include/deal.II/base/exceptions.h

```diff
@@ -194,6 +205,12 @@ class ExceptionBase : public std::exception
  * the exception, it simply reverts to the default text provided when
  * declaring the exception class through this macro.
  *
+ * @note This and similar macro names are examples of preprocessor definitions in the deal.II library that are not prefixed by a string that likely makes them unique to deal.II. As a consequence, it is possible that other libraries your code interfaces with defines similar names, and the result will be name collisions (see https://en.wikipedia.org/wiki/Name_collision). One can <code>#undef</code> this macro, as well as all other macros
```

Comment:
I think you want to break this line. Same below.

Reply:
fixed